packages/react-aria-components/test/Breadcrumbs.test.js

```diff
@@ -103,4 +103,17 @@ describe('Breadcrumbs', () => {
     let item = getByRole('listitem');
     expect(breadcrumbRef.current).toBe(item);
   });
+
+  it('should support autoFocusCurrent', () => {
+    let {getAllByRole} = render(
+      <Breadcrumbs autoFocusCurrent>
+        <Breadcrumb><Link href="/">Home</Link></Breadcrumb>
+        <Breadcrumb><Link href="/react-aria">React Aria</Link></Breadcrumb>
+        <Breadcrumb><Link href="/react-aria">useBreadcrumbs</Link></Breadcrumb>
+      </Breadcrumbs>
+    );
+
+    let links = getAllByRole('link');
+    expect(links[2]).toHaveFocus();
```

Comment:
oo, nice, didn't know about this matcher. we can replace `expect(document.activeElement).toBe(links[2])` everywhere eventually

Reply:
That would be an easy ast-grep codemod:

```yaml
rule:
  pattern: expect(document.activeElement).toBe($A)
fix: expect($A).toHaveFocus()
```

I'm actually curious if that matcher is any faster. If so, we could probably speed up our suite a good bit, as much as we check for focus.

Reply:
Just tried locally, didn't clear cache or do any other isolation measures though:

Before:
```
Time:        192.227 s
```

After:
```
Time:        160.324 s
```

Reply:
hmm i wouldn't think it would be faster, document.activeElement is just an access to something already up to date, and then followed by an identity check. I imagine it's what the matcher is doing behind the scenes as well. ok, yep, it is, https://github.com/testing-library/jest-dom/blob/f03a582827453de6fac4510dcf4fa3148c7ed68a/src/to-have-focus.js#L7
so I'm betting you're seeing caching of some sort